web/frontend/src/components/navigation/index.svelte

```diff
@@ -132,6 +132,7 @@ const handleDeleteWaypoint = async (event: CustomEvent<string>) => {
           [navigationApi.Mode.MODE_UNSPECIFIED]: '',
           [navigationApi.Mode.MODE_MANUAL]: 'Manual',
           [navigationApi.Mode.MODE_WAYPOINT]: 'Waypoint',
+          [navigationApi.Mode.MODE_EXPLORE]: 'Explore',
```

Comment:
@DTCurrie not sure how Ethan's PR this morning passed because mine is failing listing without this change is it fine to just add this here?

Reply:
@ethanlook do you have any thoughts/ideas on this as well?

Reply:
My PR didn't actually bump the TS-SDK version but this is also an issue. This is safe to add since explore mode is not one of the options above.